src/api/plugins/tdbp/warren_tdbp/exceptions.py

```diff
@@ -0,0 +1,11 @@
+"""Exceptions for TdbP."""
+
+
+class ConfigurableIndicatorConsistencyException(Exception):
```

Comment:
```suggestion
class IndicatorConsistencyException(Exception):
```
Name may be a bit long?